app/views/shared/_sorting.html.erb

```diff
@@ -0,0 +1,36 @@
+<%#
+   "Renders a sorting widget
+
+    Variables:
+      types : Array<String> of available sorting types
+    ? default_order : Defualt sorting order, if any
+    ? default_type  : Default sorting type, if any
+"%>
+
+<%
+  # Defaults
+  default_order = defined?(default_order) ? default_order : nil
+  default_type = defined?(default_type) ? default_type : nil
+%>
+
+<div class="flex gap-sm">
+  <div class="button-list is-gutterless">
+    <% [:asc, :desc].each do |order| %>
+      <% is_active = params[:order]&.to_sym == order || (params[:order].nil? && default_order == order) %>
+      <%= link_to order.to_s.humanize, request.params.merge(order: order),
```

Comment:
For sorting icons, you should have `fa-sort`, `fa-sort-up`, and `fa-sort-down` available.